src/lib/mina_networking/mina_networking.mli

```diff
@@ -109,6 +109,12 @@ module Rpcs : sig
       option
   end
 
+  module Get_completed_snarks : sig
+    type query = unit
+
+    type response = Transaction_snark_work.Checked.Stable.V2.t list option
```

Comment:
Remove `Checked`? `Checked` modules are to signal that the value has been verified/checked, within a node. Across the network, nodes will/should verify responses upon receipt regardless